dvmkv2mp4

```diff
@@ -479,10 +526,17 @@ EOF
   coment_tag=""
   echo "-------------"
 done
+
+rm "${LOCKFILE}"
+
+if [ "$TEMP_DIR_CREATED" = 'yes' ]; then
```

Comment:
this should be moved to debug section, in case of debug we want to keep temp files to be able to debug conversion